echopype/tests/conftest.py

```diff
@@ -1,13 +1,47 @@
 """``pytest`` configuration."""
 
 import pytest
-from pathlib import Path
 
 import fsspec
 
 from echopype.testing import TEST_DATA_FOLDER
 
 
+def pytest_generate_tests(metafunc):
+    ek80_new_path = TEST_DATA_FOLDER / "ek80_new"
+    ek80_files = ek80_new_path.glob("**/*.raw")
+    if "ek80_file" in metafunc.fixturenames:
+        metafunc.parametrize(
+            "ek80_file", ek80_files, ids=lambda f: str(f.name)
+        )
+
+
+@pytest.fixture(scope="session")
+def ek80_file(request):
+    return request.param
```

Comment:
Since `ek80_file` is only used in `test_convert_ek80.py`, it seems that these can be moved to that file? 

Also, could you change `ek80_file` to `ek80_new_file` just so that it is clear which set of files are being referred to here?